Projects/Core/Models/GroupDetail/MemberList.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  MemberList.swift
+//  CoreKit
+//
+//  Created by 최혜린 on 6/22/24.
+//  Copyright © 2024 com.mashup.gabbangzip. All rights reserved.
+//
+
+import Foundation
+
+public struct Member: Equatable, Identifiable {
+  public var id = UUID()
+  
+  public let name: String
+  public let isLeader: Bool
+  
+  public init(
+    name: String,
+    isLeader: Bool
+  ) {
+    self.name = name
+    self.isLeader = isLeader
+  }
+  
+  public static var leaderMock: Member {
+    Member(
+      name: "혜린",
+      isLeader: true
+    )
+  }
+  
+  public static var notLeaderMock: Member {
+    Member(
+      name: "혜린",
+      isLeader: false
+    )
+  }
+  
+  public static var mockList: [Member] {
+    [
+      Member(
+        name: "혜린",
+        isLeader: true
+      ),
+      Member(
+        name: "현아",
+        isLeader: false
+      ),
+      Member(
+        name: "준혁",
+        isLeader: false
+      )
+    ]
+  }
+}
+
+public typealias MemberList = [Member]
```

Comment:
P5: typealias를 맨 위에 위치시키는건 어떤가요?

Reply:
```suggestion
public typealias MemberList = [Member]

public struct Member: Equatable, Identifiable {
```

이런 식으로 Member 선언 전에 말씀하시는거죠? 혹시 이유가 있을까요 ?!

Reply:
typealias로 지정한 타입을 찾기 위해 해당 파일로 들어왔을 때 맨 위에 있는 것이 찾기 편할거 같다고 생각해서요! 하지만 현재 tyealias의 경우 User-Defined 타입이기 때문에 Member타입 정의가 완료된 후 선언해야한다고 생각하신다면 그것도 충분히 설득력 있다고 생각해용

Reply:
저는 말씀하신 이유 때문에 뒤 쪽에 배치하긴 했습니다! 타입 찾으러 갈 때 command + 클릭으로 찾아가면 어차피 해당 타입 선언된 부분으로 포커스 이동까지 돼서 지금 상태로 두는 것도 나쁘지 않을 것 같다는 의견입니다 😄

Reply:
![스크린샷 2024-07-04 오후 8 42 31](https://github.com/mash-up-kr/gabbangzip-iOS/assets/101619749/716c6901-7f10-42a2-8ed0-a80d867612e4)
진행시키시죠